lib/tasks/aact.rake

```diff
@@ -3,4 +3,20 @@ namespace :aact do
     updater = Util::Updater.new
     updater.start
   end
+
+  task :delete_previous_month_files, [] => :environment do
+    last_month_start = Date.today.last_month.beginning_of_month
+    last_month_end = Date.today.last_month.beginning_of_month.end_of_month+1
+    files = FileRecord.where(file_type: 'snapshot').
```

Comment:
I think we talked about passing a parameter that specifies the type of file to delete.